txstratum/manager.py

```diff
@@ -52,6 +50,11 @@ def __init__(self, backend: 'HathorClient'):
         self.block_template_updated_at: float = 0
         self.block_template: Optional['BlockTemplate'] = None
 
+        # Statistics
+        self.txs_solved: int = 0
+        self.blocks_found: int = 0
+        self.txs_timeout: int = 0
```

Comment:
Not right now but I think we could create a task to add this info to prometheus, so we can add this to grafana if we want. 

But maybe we could have this on a `status` API

Reply:
I added them to the `status` API. I will also add these metrics to our Grafana in the next PR. I'll focus on deployment and monitoring.